src/js/themes/base.js

```diff
@@ -1036,6 +1040,59 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // colors: [] || colors: ['graph-0', 'graph-1', 'graph-2', 'graph-3'],
       // extend: undefined,
     },
+    notification: {
+      toast: {
+        container: {
+          gap: 'small',
+          elevation: 'medium',
+          round: 'small',
+        },
+      },
+      iconContainer: {
+        pad: 'small',
+        round: { size: 'small', corner: 'left' },
+      },
+      textContainer: {
+        pad: { left: 'small', right: 'medium', vertical: 'small' },
+      },
+      messageText: {
+        size: 'large',
+        weight: 'bold',
+        color: 'black',
+      },
+      bodyText: {
+        size: 'medium',
+        margin: 'none',
+        color: 'black',
+      },
+      button: {
+        margin: { left: 'medium' },
+        icon: FormClose,
+      },
+      critical: {
```

Comment:
Wondering if it would make sense for all of these status related objects to be nested under a `status` key.

```
status: {
   critical: {
     ...
    },
   warning: {
    ...
    },
    ...
}
```

Reply:
i had it under a status key, but changed it based off of previous comments from @ericsoderberghp for consistency with other theming. thoughts on changing this @ericsoderberghp?